spec/unit/token_spec.rb

```diff
@@ -17,4 +17,33 @@
       end
     end
   end
+
+  describe '.load' do
+    context 'token file exists' do
+      it 'reads the file, instantiates and returns a token' do
+        expected_token_value = 'foo token 1234'
+
+        Typekitter::Token.save(expected_token_value)
+        token = Typekitter::Token.load
+
+        expect(token.value).to eq(expected_token_value)
+      end
+    end
+  end
+
+  describe '.token_exists?' do
+    context 'token file does not exist in the filesystem' do
+      it 'returns false' do
+        expect(Typekitter::Token.token_exists?).to eq(false)
+      end
+    end
+
+    context 'a token file exists in the filesystem' do
+      it 'returns true' do
+        Typekitter::Token.save('foo bar token')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.